test/fishjam/cluster/load_balancing_test.exs

```diff
@@ -53,11 +62,98 @@ defmodule Fishjam.Cluster.LoadBalancingTest do
     assert_rooms_number_on_fishjam(node2, 1)
   end
 
+  describe "peer websocket load balancing" do
+    setup do
+      [node1, _node2] = @nodes
+
+      response_body = add_room(node1)
+      assert %{"id" => room_id} = response_body["data"]["room"]
+      fishjam_instance = get_fishjam_address(response_body)
+
+      response_body = add_peer(fishjam_instance, room_id)
+      token = response_body["data"]["token"]
+      peer_id = get_in(response_body, ["data", "peer", "id"])
+      other_node = @nodes |> Enum.find(&(&1 != fishjam_instance))
+
+      on_exit(fn ->
+        HTTPoison.delete("http://#{fishjam_instance}/room/#{room_id}", @headers)
+      end)
+
+      {:ok,
+       %{
+         room_id: room_id,
+         token: token,
+         room_instance: fishjam_instance,
+         other_instance: other_node,
+         peer_id: peer_id
+       }}
+    end
+
+    @tag timeout: @max_test_duration
+    test "delete room on other node", state do
```

Comment:
I think this one and one on `99` tests HTTP API not the WS since you are setting up ws and then make a request to controller not a ws message.

Reply:
It's testing that the peer socket is correctly closed on another node when the room is another node, and it is deleted. (So it is testing the logic forwarding information about deleting peer/room from one node to another).